autograder/rest_api/views/oauth2callback.py

```diff
@@ -130,6 +130,51 @@ def load_user_info(self, request):
             raise
 
 
+class AzureOAuth2CallbackHandler(OAuth2CallbackHandler):
+    def load_user_info(self, request):
+        response = None
+        content = None
+        try:
+            credentials = client.credentials_from_clientsecrets_and_code(
+                settings.OAUTH2_SECRETS_PATH, GOOGLE_API_SCOPES, request.GET,
```

Comment:
I added an `AZURE_API_SCOPES` var in auth.py with my recent changes. Can we replace `GOOGLE_API_SCOPES` with that here?

Reply:
I have been reading over your scope changes. This all looks good and I will update my code to this. 

One thing, on line 34 of autograder/rest_api/auth.py you still reference GOOGLE_API_SCOPES. Should this be self.scopes?

Reply:
Yes it should, good catch! I just pushed a commit with that change